jest.config.js

```diff
@@ -30,5 +30,7 @@ module.exports = {
 
   transformIgnorePatterns: [
     "/node_modules/(?!(string-width|cliui)/)"
-  ]
+  ],
+
+  roots: ['src']
```

Comment:
required'ish: 
this isn't breaking but lets keep the "alphabetical" sorting of the properties, so move it up around line `27` ahead of `testEnvironment`